plenum/common/signer_did.py

```diff
@@ -33,11 +47,22 @@ def identifier(self) -> Identifier:
 
     @property
     def verkey(self) -> str:
+        if self._verkey is None:
+            return None
+
         if self.abbreviated:
             return self.abbr_prfx + self._verkey
         else:
             return self._verkey
 
+    def full_verkey(self):
```

Comment:
You should make this a property if you don't intend to pass any extra arg like option to return raw of friendly